File: ML/Saksham-Gupta-30_SakshamGupta_2024csit1164_2/GettingStarted/PrintAllPrimesTillN.cpp
#include <iostream>
#include <cmath>
using namespace std;

int isPrime(int n)
{
    int i;
    for (i = 2; i < sqrt(n); i++)
    {
        if (n % i == 0)
            return 0;
    }
    return 1;
}

int main()
{
    int i, low, high;
    cin >> low >> high;
    for (i = low; i <= high; i++)
    {
        if (isPrime(i) == 1)
            cout << i << endl;
    }
}

File: Android/aashishkoder24_aashishraghav_2024IT1010_2/GettingStarted/PrintAllPrimesTillN.java
import java.util.*;
import java.math.*;

public class Main{
    public static void main(String[] args) {
        
        Scanner sc=new Scanner(System.in);
      
        int low=sc.nextInt();
        int high=sc.nextInt();
        if(low!=1)
        {
        
      for(int i=low;i<=high;i++)
      {
        int o= isprime(i);
         
        if(o==1)
        System.out.println(i);
      }
        }
        else
        System.out.println(1);
    }
    static int isprime(int l)
     {
         int c=0;
         
          for(int j=1;j<=(int)Math.floor(Math.sqrt(l));j++)
        {
            if(l%j==0)
            c++;
        }
        return c;
     }
      
      
        
    
}